parseAndPopulate/runCapabilities.py

```diff
@@ -184,25 +191,26 @@ def main(scriptConf=None):
         for pattern in patterns:
             for filename in find_files(args.dir, pattern):
                 update = True
+                # NOTE: Hash of file will be saved for each file individually, not only capability xml files
                 if not args.api and args.save_modification_date:
+                    files_modifications = {}
+                    path = '{}/filesModifications.json'.format(temp_dir)
```

Comment:
Very good point, I did not think about the meaning of this folder in this way, I agree to move this file elsewhere.
However, this functionality will be redesigned during the implementation of [this functionality](https://github.com/YangCatalog/backend/issues/183). Please, think of this as a demo of the larger functionality that will be added soon. Do you agree that this will be changed later?

Reply:
ok so just keep this in mind when implementing that issue

Reply:
ok, noted.